### DISCUSSION TITLE
How to remove erroneous data from db via android app (nightly)

### DISCUSSION THREAD

**Discussion by SimianSilverback** [NightscoutFoundation/xDrip#2068] [Q&A] [answered]
Hi, Is there a way of deleting erroneous data form the Android app? 

I generally get high readings until calibrated after a restart. (also with new sensors). 

I would like to remove these pre-calibration readings. (actually a number of data points)

It's been mentioned that the data can be marked as bad but I'm unable to see an obvious way to do this.

**Comment by Navid200** [comment, marked as answer]
Enable:
 Settings &#8722;> Less common settings &#8722;> Show Datables

Now, you will see two new options under the top left menu button on the main screen.  One of them is BG Data Table.  Tap on it.
You need to scroll through to get to the point that is incorrect.  Long press it.  You will have the option of marking it bad.

**Reply by SimianSilverback** [reply]
Many thanks

**Reply by vforvilela** [reply]
There's a table for "glucose" and another for "calibration".
What about carbs?

I accidentally entered a wrong carbs value and need to remove it.

**Reply by Navid200** [reply]
@vforvilela If you entered the treatment less than 24 hours ago, you can delete it:
https://navid200.github.io/xDrip/docs/Delete_Treatment.html
